### PR TITLE
support rename all file for go.mod project

### PR DESCRIPTION
**Pull request by lvht**
gopls rename require to switch to the root dir to rename all files.

### CODE REVIEW

File: autoload/go/rename.vim [hunk 1]
```diff
@@ -52,7 +52,12 @@ function! go#rename#Rename(bang, ...) abort
     return
   endif
 
-  let [l:out, l:err] = go#util#ExecInDir(l:cmd)
+  let l:wd = go#util#ModuleRoot()
+  if l:wd == -1
+    let l:wd = getcwd()
```
Comment (bhcleek) [inline autoload/go/rename.vim:57]: `l:wd` will need to be set to the directory of the current buffer here for parity with current behavior.

**Comment by lvht** [comment]
@bhcleek

**Comment by lvht** [comment]
> are you ok if I push to your branch
feel free to make any change @bhcleek

**Comment by bhcleek** [comment]
Apologies for force pushing to your branch. I've only force pushed over the new commits that I've added, though, so your local branch shouldn't be affected. I'll merge as soon as tests pass.

**Pull request merged**